docker/try-tooljet.Dockerfile

```diff
@@ -5,29 +5,30 @@ COPY --from=postgrest/postgrest:v10.1.1.20221215 /bin/postgrest /bin
 
 # Install Postgres
 RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
-RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main 13" | tee /etc/apt/sources.list.d/pgdg.list
+RUN echo "deb http://deb.debian.org/debian"
```

Comment:
this isn't required?

Reply:
Yep so that it add the required debian dependence which were throwing error in docker 

2023-02-27T06:48:23Z #12 4.233 postgresql-13 : Depends: libicu63 (>= 63.1-1~) but it is not installable
2023-02-27T06:48:23Z #12 4.235 Depends: libllvm7 (>= 1:7~svn298832-1~) but it is not installable
2023-02-27T06:48:23Z #12 4.235 Recommends: sysstat but it is not going to be installed
2023-02-27T06:48:23Z #12 4.236 postgresql-client-13 : Depends: libreadline7 (>= 6.0) but it is not installable

Reply:
Ohh understood